projects/com.oracle.truffle.llvm.nativeint/src/com/oracle/truffle/llvm/nativeint/NativeLookup.java

```diff
@@ -93,7 +93,7 @@ public static NativeFunctionInterface getNFI() {
     }
 
     private static List<NativeLibraryHandle> getNativeFunctionHandles() {
-        String[] dynamicLibraryPaths = LLVMBaseOptionFacade.getDynamicLibraryPaths();
+        String[] dynamicLibraryPaths = LLVMOptions.ENGINE.getDYN_LIBRARY_PATHS();
```

Comment:
yes sure, @Option has a name property that we can set to provide better names.
I will refactor my changes and use this feature.